pages/about.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import Layout from '../components/Layout';
+import { Col } from 'reactstrap';
+
+export default class IndexPage extends React.Component {
+  render() {
+    return (
+      <div>
+        <Layout title="Vision Zero Boston" aboutPage>
+          <Col className="pl-5 pr-5">
+            <div className="sh mt-4 mb-4">
+              <h2 className="sh-title">Vision Zero Crash Records Map</h2>
+            </div>
+            <p>
+              <a href="http://visionzeroboston.org">Vision Zero Boston</a> is
+              our commitment to focus the city’s resources on proven strategies
+              to eliminate fatal and serious traffic crashes in the city by
+              2030. We are inspired by the belief that even one fatality is too
+              many.
+            </p>
+            <p>
+              This map and associated dataset, provided as part of the Vision
+              Zero Boston program, contains records of the date, time, location,
+              and type of crash for incidents requiring public safety response
+              which may involve injuries or fatalities. All records are compiled
+              by the Department of Innovation and Technology from the City’s
+              Computer-Aided Dispatch (911) system and verified as having
+              required a response from a public safety agency. To protect the
+              privacy of individuals involved in these incidents, we do not
+              indicate the severity of specific crashes or whether medical care
+              was provided in any specific case.{' '}
```

Comment:
Can remove this `{' '}` and I think a few others.

Reply:
👍